pkg/kube/client.go

```diff
@@ -157,6 +157,9 @@ type Client interface {
 
 	// Shutdown closes all informers and waits for them to terminate
 	Shutdown()
+
+	// ClusterID returns the cluster this client is connected to
+	ClusterID() cluster.ID
```

Comment:
Why do we need to know clusterID from k8s client, we can get it from controller already